gems/manageiq_foreman/manageiq_foreman.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |spec|
   spec.email         = ["keenan@thebrocks.net"]
   spec.description   = %q{Foreman Provider for ManageIQ}
   spec.summary       = %q{Foreman Provider for ManageIQ}
-  spec.homepage      = "http://github.com/manageiq/manageiq_foreman"
+  spec.homepage      = "https://github.com/ManageIQ/manageiq"
```

Comment:
Is the longer term idea to split out into its own repo?
grep would be harder, travis would be easier, code climate would be easier.
Not suggesting we push to rubygems.org, we could submodule or git checkout the file into the repo.